libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -226,6 +226,12 @@ void AP_DDS_Client::populate_static_transforms(tf2_msgs_msg_TFMessage& msg)
         msg.transforms[i].transform.translation.y = -1 * offset[1];
         msg.transforms[i].transform.translation.z = -1 * offset[2];
 
+        // Ensure rotation is normalised
```

Comment:
This is really "initialised", not "normalised", right?  IIUC these *would* be normalised, it's just not their only normal form.

Reply:
https://github.com/ArduPilot/ardupilot/pull/27161

Yep. Common function and rename here ^